recipes/jsonnet/all/conanfile.py

```diff
@@ -51,6 +52,11 @@ def validate(self):
         if self.settings.compiler.cppstd:
             tools.check_min_cppstd(self, "11")
 
+        if self.options.shared and \
+           ((str(self.settings.compiler) == "Visual Studio" and self.settings.compiler.runtime in ["MTd", "MDd"]) or \
+            (str(self.settings.compiler) == "msvc" and self.settings.compiler.runtime in ["MT", "MD"] and self.settings.compiler.runtime_type == "Debug")):
+            raise ConanInvalidConfiguration("shared jsonnet is not supported with MTd runtime")
```

Comment:
use `msvc_runtime_flag` instead and a helper function `_is_msvc` to keep things simple.

see https://github.com/conan-io/conan-center-index/blob/b7d8f8f9a16c2c949fe90364f1403906094dc8fb/recipes/bigint/all/conanfile.py#L51-L52

Reply:
Fixed